File: C/Listas/Lista 2/ex06.c
//6) Faça um programa que leia a idade de uma pessoa expressa em dias e mostre-a expressa em anos, meses e dias.
#include <stdio.h>

int main(){
    int dias, anos,meses,dias_rest;
    printf("Informe a quantidade de dias vividos: ");
    scanf("%d",&dias);
    anos=dias/365;
    meses=(dias%365)/30;
    dias_rest=(dias%365)%30;
    printf("A quantidade de %d será expressa da seguinte forma: ",dias);
    printf("Anos = %d\nMeses = %d\nDias = %d",anos,meses,dias_rest);
    return 0;
}

File: Java/extra/Lista 02/src/ex06.java
/* Faça um programa que leia a idade de uma pessoa expressa em dias e mostre-a 
expressa em anos, meses e dias. */

import java.util.Scanner;

public class ex06 {
    public static void main(String[] args) {
        try (Scanner scanner = new Scanner(System.in)) {
            System.out.print("Digite a idade em dias: ");
            int idadeEmDias = scanner.nextInt();

            int anos = idadeEmDias / 365;
            int meses = (idadeEmDias % 365) / 30;
            int dias = (idadeEmDias % 365) % 30;

            System.out.println("Idade em anos: " + anos + " anos");
            System.out.println("Idade em meses: " + meses + " meses");
            System.out.println("Idade em dias: " + dias + " dias");
        }
    }
}